scripts/ci_setup.py

```diff
@@ -378,6 +379,15 @@ def main(args: Any):
     else:
         dotnet.setup_dotnet(args.dotnet_path)
 
+    framework = ChannelMap.get_target_framework_moniker(args.channel)
+    if framework == 'net8.0' or framework == 'nativeaot8.0':
+        # Copy the global.json file to global.bak.json
+        global_json_path = os.path.join(get_repo_root_path(), 'global.json')
+        shutil.copy(global_json_path, os.path.join(get_repo_root_path(), 'global.net9.json'))
```

Comment:
Do we need to keep the "original" file? We could just blindly overwrite it with _net8_ file, right?